update: homepage and sponsor details (#405)
graphile/graphile.github.io: src/pages/sponsor.mdx

<Col span="6">
<MarketingProduct
big
name={<span>Support Gra<em>fast</em> and PostGraphile V5</span>}
headline="The next-generation planning and execution engine for GraphQL"
>

> Gra<em>fast</em> and PostGraphile Version 5 are now in beta; 
> this timely release was in large part due to the support given through sponsorship. 

Reviewer: ```suggestion
Gra<em>fast</em> and PostGraphile Version 5 are now in open beta thanks
in large part to the support given through sponsorship.
```

> Our sponsors enable us to continue to concentrate on our projects with the freedom to work on 
> our software in the most effective way &mdash; without diverting our resources to other endeavors. 

Reviewer: ```suggestion
Sponsorship is the most effective way to support our open source projects &mdash; it enables
us to invest time directly into their maintenance and advancement without adding additional
time burdens.
```
